webserver/external/musicbrainz.py

```diff
@@ -12,7 +12,7 @@ def get_recording_by_id(mbid):
             recording = musicbrainzngs.get_recording_by_id(mbid, includes=['artists', 'releases', 'media'])['recording']
         except ResponseError as e:
             raise DataUnavailable(e)
-    cache.set(mbid, recording, time=CACHE_TIMEOUT)
+    _redis.setex(mbid, recording, CACHE_TIMEOUT)
```

Comment:
We should use https://github.com/metabrainz/brainzutils-python/blob/master/brainzutils/cache.py